packages/react-components/src/templates/TeamProfileAbout.tsx

```diff
@@ -49,19 +57,23 @@ const TeamProfileAbout: React.FC<TeamProfileAboutProps> = ({
         projectTitle={projectTitle}
         projectSummary={projectSummary}
         proposalURL={proposalURL}
+        tags={tags}
```

Comment:
Tags end up being duplicated in the UI, see this example when forcing some tags in the component:

<img width="1856" height="1001" alt="Image" src="https://github.com/user-attachments/assets/b735261b-d22e-439e-9061-1b4ac092fb0e" />